packages/framework/src/Component/UploadedFile/UploadedFileFacade.php

```diff
@@ -195,8 +238,21 @@ public function getAbsoluteUploadedFileFilepath(UploadedFile $uploadedFile)
      * @param \Shopsys\FrameworkBundle\Component\UploadedFile\UploadedFile $uploadedFile
      * @return string
      */
-    public function getUploadedFileUrl(DomainConfig $domainConfig, UploadedFile $uploadedFile)
+    public function getUploadedFileUrl(DomainConfig $domainConfig, UploadedFile $uploadedFile): string
     {
         return $this->uploadedFileLocator->getUploadedFileUrl($domainConfig, $uploadedFile);
     }
+
+    /**
+     * @param \Shopsys\FrameworkBundle\Component\UploadedFile\UploadedFile[] $uploadedFiles
+     */
+    protected function updateFilesOrder(array $uploadedFiles): void
```

Comment:
```suggestion
    protected function saveFilesOrdering(array $uploadedFiles): void
```
Wouldn't be better to use same naming convention as `ImageFacade` does ?  Or change function name in `ImageFacade`

Reply:
Well I was thinking about this and I have decided this way because:
1) ImageFacade saves order of images that has been uploaded before this action
2) UploadedFileFacade does same plus it counts position of newly created files. I like update more.

Reply:
ok